netaddr.go

```diff
@@ -715,40 +688,33 @@ func (p IPPrefix) IPNet() *net.IPNet {
 	}
 }
 
+func mask4(n uint8) uint32 {
+	return ^uint32(0) << (32-n)
+}
+
+func mask6(n uint8) (uint64, uint64) {
+	if n > 64 {
+		return ^uint64(0), ^uint64(0) << (64-(n-64))
```

Comment:
Why `(64-(n-64))` instead of `(128-n)`?

Reply:
Because math at 2am is hard, apparently. Fixed.